modules/gapi/test/common/gapi_tests_common.hpp

```diff
@@ -88,16 +89,28 @@ class TestFunctional
 
     cv::Scalar sc;
 
+    // integral Scalar initialization
     cv::Scalar initScalarRandU(unsigned upper)
     {
-        auto& rng = cv::theRNG();
-        double s1 = rng(upper);  // FIXIT: RNG result is 'int', not double
+        cv::RNG rng(time(NULL));
```

Comment:
Why switched from the original `thRNG`? It looks like it casts to uint64 anyway. At least use `nullptr` or `0`, not `NULL`

Reply:
The original implementation of RNG generation in fact didn't use to generate RNG - it used to always start from the same number so Scalars and Mats used to be equal for all of the tests and for every run. So I decided to add the time

`NULL` -> `nullptr`, I don't know why I wrote `NULL`